participants/Gunessee/pr_tutorial/simple_functions.py

```diff
@@ -10,3 +10,10 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+    
+def is_prime(value):
+    for i in range(2,value):
```

Comment:
great idea to add this function! consider testing if the user accidentally uses a string as an input:)

Reply:
Fixed, thank you